src/docc/plugins/python/cst.py

```diff
@@ -676,7 +683,10 @@ def _assign_docstring(self) -> Optional[nodes.Docstring]:
 
         try:
             line_parent_context = self.old_stack[-2]
-            sibling_index = line_parent_context.child_offset + 1
+            if isinstance(line_parent_context.node.cst_node, cst.Module):
```

Comment:
I'm not entirely sure what this change does, but does it work for nested classes?